lib/membrane_rtsp/session/session.ex

```diff
@@ -0,0 +1,119 @@
+defmodule Membrane.RTSP.Session do
```

Comment:
As we disscussed move here all functions from `Membrane.RTSP` module and rename it from `Membrane.RTSP.Session` to `Membrane.RTSP`